### PR TITLE
Fix absent classes

### PR DESCRIPTION
**Pull request by vcarpani**

### CODE REVIEW

File: keras_retinanet/bin/evaluate.py [hunk 1]
```diff
@@ -2,13 +2,10 @@
 
 """
 Copyright 2017-2018 Fizyr (https://fizyr.com)
-
```
Comment (hgaiser) [inline keras_retinanet/bin/evaluate.py]: Why are these removed?

File: keras_retinanet/bin/evaluate.py [hunk 2]
```diff
@@ -132,23 +129,25 @@ def main(args=None):
...
-            score_threshold=args.score_threshold,
-            max_detections=args.max_detections,
-            save_path=args.save_path
-        )
-
-        # print evaluation
-        for label, average_precision in average_precisions.items():
-            print(generator.label_to_name(label), '{:.4f}'.format(average_precision))
-        print('mAP: {:.4f}'.format(sum(average_precisions.values()) / len(average_precisions)))
+    average_precisions = evaluate(
```
Comment (hgaiser) [inline keras_retinanet/bin/evaluate.py]: The way it is implemented now, `evaluate_coco` isn't used. COCO has quite a nice evaluation output, but it is unfortunately incompatible with other types of datasets. Still though, for COCO I think we should use `evaluate_coco`.

File: keras_retinanet/callbacks/eval.py [hunk 3]
```diff
@@ -67,6 +76,4 @@ def on_epoch_end(self, epoch, logs=None):
         logs['mAP'] = self.mean_ap
 
         if self.verbose == 1:
-            for label, average_precision in average_precisions.items():
-                print(self.generator.label_to_name(label), '{:.4f}'.format(average_precision))
```
Comment (hgaiser) [inline keras_retinanet/callbacks/eval.py]: Printing the mAP per label can be quite useful. Why did you remove it?

File: keras_retinanet/bin/evaluate.py (hunk 1 above)
Comment (carpo93) [reply keras_retinanet/bin/evaluate.py]: I have no idea why did it happened, will check

File: keras_retinanet/callbacks/eval.py (hunk 3 above)
Comment (vcarpani) [reply keras_retinanet/callbacks/eval.py]: It is not removed, I just moved it inside the for loop at line 60

Comment (hgaiser) [reply keras_retinanet/callbacks/eval.py]: I see, then I haven't said anything :)

**Comment by hgaiser** [comment]
It seems there are some unit test errors. If they're addressed, it LGTM.
